### PR TITLE
Fix split view expand/collapse button

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13276
affects: student
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/Core/Extensions/UIImageExtensions.swift [hunk 1]
```diff
@@ -24,6 +24,7 @@ extension UIImage {
         case attendance, cameraSolid, collaborations, conferences
         case todo, todoSolid
         case addAudioLine, addCameraLine, addDocumentLine, addImageLine, addVideoCameraLine
+        case splitCollapse, splitExpand
```
Comment (armstrongnate) [inline Core/Core/Extensions/UIImageExtensions.swift]: I looked in inst-ui for these but they don't have exactly what we want.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1006870&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Core/Core/Extensions/UIViewControllerExtensions.swift [hunk 2]
```diff
@@ -55,6 +55,16 @@ extension UIViewController {
         return false
     }
 
+    public var displayModeButtonItem: UIBarButtonItem? {
+        guard let splitView = splitViewController else { return nil }
+        let defaultButton = splitView.displayModeButtonItem
```
Comment (vanwagonet) [inline Core/Core/Extensions/UIViewControllerExtensions.swift]: Does the default appearance not look right so we need to use the custom icons?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-13276](https://instructure.atlassian.net/browse/MBL-13276)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.24% | 93.24% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Extensions/UIViewControllerExtensions.swift (hunk 2 above)
Comment (armstrongnate) [reply Core/Core/Extensions/UIViewControllerExtensions.swift]: Yeah, and it doesn't have a "collapse" state. It's really weird. I don't like that we have to customize it but what we have makes sense. I tried really hard to stick to defaults but it's pretty different from what we have.

**Comment by armstrongnate** [comment]
Just realized I need to add tests for router.

File: Core/Core/Extensions/UIViewControllerExtensions.swift (hunk 2 above)
Comment (vanwagonet) [reply Core/Core/Extensions/UIViewControllerExtensions.swift]: sounds good.

**Pull request merged**